lib/commands/create.js

```diff
@@ -41,7 +46,12 @@ module.exports = function (program) {
       util.log('create', `Adding your project shortname (${shortName}) [2/4]`);
 
       // Do some token replaces to rename the project
-      replaceInConfig(/a3-boilerplate|apostrophe-boilerplate|apostrophe-open-museum/g, shortName);
+      // replaces the shortname in app.js
+      replaceInConfig(/(shortName:).*?,/gi, `$1 '${shortName}',`);
+      // replaces the shortname and repository URL in package.json
+      replaceInConfig(/("name":).*?,/g, `$1 "${shortName}",`);
+      replaceInConfig(/("url":)\s?"[^"]*"/g, `$1 "${newUrl}"`);
```

Comment:
OK, so `newUrl` is supposed to be the location in your own github for your own new project. But we can't possibly know that. It definitely won't be in Apostrophe's github org, and we don't know what the user's org is called, or if they want github. This may be a very old problem, or just one we should refrain from introducing now.

Reply:
This was a combination of me trying to rush too much and introducing the `@` and a replication of existing "functionality". I agree that modifying the original template repo in the `package.json` doesn't make sense and should be eliminated.